### DISCUSSION TITLE
[Feature] Integration of Shared pictures

### DISCUSSION THREAD

**Discussion by nicoloma89** [immich-app/immich#10735] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be nice if a user could integrate shared pictures into their own albums.

For example, I was at a party with a friend and I made a video of his Children with my mobile phone. Now he wants that video. I shared the video with him. But i dont need and want that video. But, he cannot integrate it into his own albums.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1587

**Reply by nicoloma89** [reply]
sorry, i have searched... but not good anough... sorry
